packages/analytics/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    parserOptions: {
+        tsconfigRootDir: __dirname,
```

Comment:
Every package needs to have this ti make sure the eslint uses the local `tsconfig` and not the global one as most of packages has some custom configurations.